doc/related_projects.rst

```diff
@@ -184,6 +184,12 @@ and tasks.
   K-means and mixture of von Mises Fisher clustering routines for data on the
   unit hypersphere.
 
+**Dataset resampling**
```

Comment:
I think this is the wrong heading... ultimately this stuff is (almost) entirely for classification; does it belong there? Otherwise unsure what the right heading is.